sale_order_invoicing_queued/models/queue_job.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models
+
+
+class QueueJob(models.Model):
+    _inherit = 'queue.job'
+
+    def cancel_sudo(self):
```

Comment:
It seems to me like this should go directly in the queue addon, isn't it? This doesn't seem to be specific for invoices...

Reply:
No, upstream already has `unlink` functionality. This is only a bridge for being executed through a button.